attributes/network/cddl/2_ledger.cddl

```diff
@@ -10,49 +10,119 @@ ledger-symbol = address
 ; for simplicity. The networks CAN use either when serializing, but
 ; MUST accept both when doing transactions.
 ledger-amount = biguint / uint
-; end::types[]
 
-; tag::info[]
-; Info endpoint.
-ledger.info@param = ()
-ledger.info@return = (ledger-info)
+; A single logo that can be shown for representing the token. A token can
+; have a set of logos that can represent visually its identity.
+single-visual-token-logo = single-visual-token-logo-unicode
+                         / single-visual-token-logo-image
 
-ledger-info = {
-    ; List of supported symbols for this ledger.
-    0 => [* ledger-symbol],
+; A unicode logo.
+; For Bitcoin this would be U+20BF ("₿").
+single-visual-token-logo-unicode = {
+    ; Type.
+    0 => 0,
 
-    ; Hash value of the ledger state. If two ledgers have the same hashes,
-    ; chances are they have the same balances on the same accounts (with
-    ; some risk of collision).
-    ; This can include configuration, depending on the implementation.
-    ; The actual hashing method and size of the hash is left to the
-    ; implementation.
-    1 => bstr,
+    ; Unicode code point. This isn't limited to a single glyph but can be
+    ; truncated or transformed by the server or the frontend.
+    1 => tstr,
+}
 
-    ; Transfer fees for each symbols, between accounts.
-    ; TODO: not implemented.
-    2 => {
-        ledger-symbol => transaction-fee,
-    },
+; A binary image.
+single-visual-token-logo-image = {
+    ; Type.
+    0 => 1,
 
-    ; Conversion rates and fees between symbols.
-    ; TODO: not implemented.
-    3 => {
-        [ledger-symbol, ledger-symbol] => conversion,
-    },
+    ; Content-type, following the IANA list. This should be part of the
+    ; image section.
+    ; See https://www.iana.org/assignments/media-types/media-types.xhtml#image
+    1 => tstr,
 
-    ; Local human readable names for symbols.
-    ; These names are local only, and are only guaranteed to be unique on this
-    ; specific ledger.
-    4 => {
-        ; TODO: this.
-        ledger-symbol => {
-            0 => tstr,
-            1 => tstr,  ; Description
-            2 => uint, ; Number of precision digits
-            tstr => tstr ; metadata
-        }
-    }
+    ; Binary content of the image.
+    2 => bstr,
+
+    ; Optional height and width, in pixels.
+    ? 3 => [ uint, uint ],
```

Comment:
Those can be extracted from the image header.

Reply:
It can actually be different in some cases (pixels vs points). https://blog.fluidui.com/designing-for-mobile-101-pixels-points-and-resolutions/

Maybe I shouldn't dig too much into that.

Reply:
I wouldn't. Let the front end deal with the data.